frontend/integration-tests/tests/kubevirt/mocks.ts

```diff
@@ -266,6 +267,40 @@ export const multusNetworkInterface = {
   networkDefinition: multusNad.metadata.name,
 };
 
+export const networkBindingMethod = {
+  pod: 'masquerade',
```

Comment:
I think it would be more clear to name the key by their value

Reply:
I updated it by using the same string for the key and the value.